### PR TITLE
Localize ConVars

### PR DESCRIPTION
**Pull request by saibotk**
This commit localizes convars, where possible.

### CODE REVIEW

File: gamemodes/terrortown/gamemode/client/cl_keys.lua [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 local timer = timer
 local IsValid = IsValid
+local cv_sv_cheats = GetConVar("sv_cheats")
```
Comment (Alf21) [inline gamemodes/terrortown/gamemode/client/cl_keys.lua]: Does it already exists there?

File: gamemodes/terrortown/gamemode/server/sv_main.lua [hunk 2]
```diff
@@ -109,11 +109,11 @@ local confirm_team = CreateConVar("ttt2_confirm_team", "0", {FCVAR_NOTIFY, FCVAR
 CreateConVar("ttt2_confirm_killlist", "1", {FCVAR_NOTIFY, FCVAR_ARCHIVE})
 
 -- innos min pct
-CreateConVar("ttt_min_inno_pct", "0.47", {FCVAR_NOTIFY, FCVAR_ARCHIVE}, "Minimum multiplicator for each player to calculate the minimum amount of innocents")
-CreateConVar("ttt_max_roles", "0", {FCVAR_NOTIFY, FCVAR_ARCHIVE}, "Maximum amount of different roles")
-CreateConVar("ttt_max_roles_pct", "0", {FCVAR_NOTIFY, FCVAR_ARCHIVE}, "Maximum amount of different roles based on player amount. ttt_max_roles needs to be 0")
-CreateConVar("ttt_max_baseroles", "0", {FCVAR_NOTIFY, FCVAR_ARCHIVE}, "Maximum amount of different baseroles")
-CreateConVar("ttt_max_baseroles_pct", "0", {FCVAR_NOTIFY, FCVAR_ARCHIVE}, "Maximum amount of different baseroles based on player amount. ttt_max_baseroles needs to be 0")
+local cv_tttt_min_inno_pct = CreateConVar("ttt_min_inno_pct", "0.47", {FCVAR_NOTIFY, FCVAR_ARCHIVE}, "Minimum multiplicator for each player to calculate the minimum amount of innocents")
```
Comment (Alf21) [inline gamemodes/terrortown/gamemode/server/sv_main.lua]: Nanana, tttt, what's that? ^^

File: gamemodes/terrortown/gamemode/server/sv_main.lua [hunk 3]
```diff
@@ -1320,9 +1320,7 @@ end
 
 local function GetEachRoleCount(ply_count, role_type)
 	if role_type == INNOCENT.name then
-		local min_innos = GetConVar("ttt_min_inno_pct")
-
-		return min_innos and math.floor(ply_count * min_innos:GetFloat()) or 0
+		return math.floor(ply_count * cv_tttt_min_inno_pct:GetFloat()) or 0
```
Comment (Alf21) [inline gamemodes/terrortown/gamemode/server/sv_main.lua]: tttt, scrrra

File: gamemodes/terrortown/gamemode/server/sv_voice.lua [hunk 4]
```diff
@@ -18,7 +18,7 @@ function MuteForRestart(state)
 end
 
 -- Communication control
-CreateConVar("ttt_limit_spectator_voice", "1", {FCVAR_NOTIFY, FCVAR_ARCHIVE})
+cv_ttt_limit_spectator_voice = CreateConVar("ttt_limit_spectator_voice", "1", {FCVAR_NOTIFY, FCVAR_ARCHIVE})
```
Comment (Alf21) [inline gamemodes/terrortown/gamemode/server/sv_voice.lua]: no localization?

File: gamemodes/terrortown/gamemode/client/cl_keys.lua (hunk 1 above)
Comment (saibotk) [reply gamemodes/terrortown/gamemode/client/cl_keys.lua]: it does work, and as this is a predefinied ConVar, i suppose it exists at start

**Pull request merged**